src/stores/ControlStore.ts

```diff
@@ -1,10 +1,6 @@
 import { writable } from 'svelte/store';
-let currentSongId = undefined;
```

Comment:
why is this not `undefined`?

Reply:
I will update so that it is initialized with empty string

Reply:
updated